physical/physical.go

```diff
@@ -88,6 +88,7 @@ var BuiltinBackends = map[string]Factory{
 	"dynamodb":  newDynamoDBBackend,
 	"etcd":      newEtcdBackend,
 	"mysql":     newMySQLBackend,
+	"postgres":  newPostgreSQLBackend,
```

Comment:
Minor aside, but given the connection string is `postgres`, is it worth calling the backend `PostgresBackend`?